Source/WebCore/platform/network/ResourceErrorBase.h

```diff
@@ -99,6 +107,7 @@ class ResourceErrorBase {
     int m_errorCode { 0 };
     Type m_type { Type::General };
     IsSanitized m_isSanitized { IsSanitized::No };
+    ErrorRecoveryMethod m_errorRecoveryMethod { ErrorRecoveryMethod::NoRecovery };
```

Comment:
We don't need additional memory to store this, and please don't add it.  We already store the domain and error code from which this can be derived.

Reply:
Add just a getter, and the non-cocoa version should probably always return ErrorRecoveryMethod::NoRecovery

Reply:
Good point, changed.